src/encoded/static/components/fetched.js

```diff
@@ -1,50 +1,42 @@
-'use strict';
-var React = require('react');
-var cloneWithProps = require('react/lib/cloneWithProps');
-var parseAndLogError = require('./mixins').parseAndLogError;
-var globals = require('./globals');
-var ga = require('google-analytics');
-var _ = require('underscore');
-
-
-var Param = module.exports.Param = React.createClass({
-    contextTypes: {
-        fetch: React.PropTypes.func,
-        session: React.PropTypes.object
-    },
-
-    getDefaultProps: function() {
-        return {type: 'json'};
-    },
-
-    getInitialState: function () {
-        return {
+import React from 'react';
```

Comment:
I think the main problem is these components aren’t documented in any way anywhere, and took me forever to understand them. `<Param>` supplies parameters to the rendering component that receives the GET request, so it all really makes sense. This is probably a good one for [Redmine 5064](http://redmine.encodedcc.org/issues/5064).